theme-data/win-system-hc/win-system-hc-common.json

```diff
@@ -143,6 +143,17 @@
           "attention": "@color-hc-PlaceholderColor",
           "secure": "@color-hc-PlaceholderColor"
         }
+      },
+      "content": {
```

Comment:
not sure if this is the correct thing to do, but the script (```npm run build:windows```) gave me errors until I added this definition in here also